tfe/resource_tfe_policy_set.go

```diff
@@ -0,0 +1,300 @@
+package tfe
+
+import (
+	"errors"
+	"fmt"
+	"log"
+	"strings"
+
+	tfe "github.com/hashicorp/go-tfe"
+	"github.com/hashicorp/terraform/helper/schema"
+)
+
+func resourceTFEPolicySet() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceTFEPolicySetCreate,
+		Read:   resourceTFEPolicySetRead,
+		Update: resourceTFEPolicySetUpdate,
+		Delete: resourceTFEPolicySetDelete,
+		Importer: &schema.ResourceImporter{
+			State: resourceTFEPolicySetImporter,
+		},
+
+		Schema: map[string]*schema.Schema{
+			"name": &schema.Schema{
+				Type:     schema.TypeString,
+				Required: true,
+			},
+
+			"description": &schema.Schema{
+				Type:     schema.TypeString,
+				Optional: true,
+			},
+
+			"global": &schema.Schema{
+				Type:     schema.TypeBool,
+				Optional: true,
+				Default:  false,
+			},
+
+			"organization": &schema.Schema{
+				Type:     schema.TypeString,
+				Required: true,
+				ForceNew: true,
+			},
+
+			"policies": &schema.Schema{
+				Type:     schema.TypeSet,
+				Required: true,
+				Elem:     &schema.Schema{Type: schema.TypeString},
+			},
+
+			"workspaces": &schema.Schema{
```

Comment:
This should be `workspace_external_ids` right? I added the `external` here because there is some unclarity what workspace ID to use in which cases. As the `workspace_id` in other resources refers to `<ORGANIZATION>/<WORKSPACE>` as all workspace related APIs expect the org and workspace name instead of the external ID.

But in here we _do_ expect the external ID right? So l hope this will make that more clear.